extensions/notebook/src/jupyter/jupyterServerManager.ts

```diff
@@ -43,7 +43,7 @@ export class LocalJupyterServerManager implements nb.ServerManager, vscode.Dispo
 	}
 
 	public get isStarted(): boolean {
-		return !!this.jupyterServer;
+		return !!this._jupyterServer;
```

Comment:
Nit. I recommend using `Boolean(${variable})` instead of the `!!`. the `!!` could cause issues depending on the type.

While writing this review I decided to fact-check myself, and it seems it does the same thing as Boolean(): https://www.sitepoint.com/javascript-double-negation-trick-trouble/